skietbaan_fe/src/index.js

```diff
@@ -5,14 +5,18 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+//import { Provider } from 'react-redux'
```

Comment:
GC - stands for Ghost Code, should be removed or Add a *TODO: reason for it to be there*